src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -212,6 +229,27 @@ public ResponseEntity<Void> cutAndPasteNode(@PathVariable("studyUuid") UUID stud
         return ResponseEntity.ok().build();
     }
 
+    @GetMapping(value = "/studies/{studyUuid}/network")
+    @Operation(summary = "check study root network existence")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "204", description = "The network does exists"),
+        @ApiResponse(responseCode = "404", description = "The network doesn't exist - Will try to reimport the network if params is set to true")})
+    public ResponseEntity<Network> getStudyNetwork(@PathVariable("studyUuid") UUID studyUuid,
```

Comment:
If I understood well, later on, it should not check if the study root exist but if the study is archived, no ? Maybe add a TODO to say this endpoint is temporary ?

Reply:
changed, this endpoint now only deal with network existence